### ISSUE TITLE
 Code snippet "Choice" feature no longer working

### ISSUE THREAD

**Issue by geoffw123** [microsoft/vscode#147950]
Sorry to have to open a bug report for an issue that was previously closed.
See Code snippet "Choice" feature no longer working #144362 

Unfortunately the Choice snippet feature is still broken for me, using the choice syntax used to work fine for ages but now I get "No Suggestions" instead of the optional choices. I have changed nothing in my snippets file, or changed any other settings
I am using today's insiders build 1.67.0

Thanks for any assistance

**Comment by geoffw123** [comment]
I dont know what other information you want me to supply about this issue.

Its OK if you dont want to acknowledge it or fix it no problem,  I just thought you might want to know its still broken and might want to fix it.

**Comment by jrieken** [comment]
https://github.com/microsoft/vscode/wiki/Submitting-Bugs-and-Suggestions#writing-good-bug-reports-and-feature-requests

**Comment by geoffw123** [comment]
Just to let you know I wont be reporting any further bugs I spot in Visual Studio  Code, due to the very poor communications I experienced with this.. Just pasting a one liner to a generic link instead of a conversation with a human being indicating why the issue was closed and telling me what further information you want me to provide. I am surprised by this I would expect Microsoft would be so much better at communications.

The irony of all this is the specific bug that I reported (repeat of issue #144362) has just been fixed in the daily build I downloaded today.

No doubt no one will be reading this or bothered at all but just thought I would let you know.
